scripts/classes/app-configurator.js

```diff
@@ -173,6 +173,25 @@ class AppConfigurator {
     return spawn('node', [reactNativeCli, 'link'], { stdio: 'inherit', cwd: process.cwd() });
   }
 
+  /**
+   * Copies files with fixes before official release
+   * With each react-native version check fixes/fixes.json and verify if issue is resolved
+   */
+  applyReactNativeFixes() {
```

Comment:
could you extract this into a new file? The idea is to call this within every native build so the fixes are applied to even if user do `npm install`.